tools/module/generate.py

```diff
@@ -127,17 +130,18 @@ def generate_cpp(parts):
         """\
         #include "{className}.hpp"
 
+        #include "extension/Behaviour.hpp"
         #include "extension/Configuration.hpp"
 
         namespace {namespace} {{
 
         using extension::Configuration;
 
-        {className}::{className}(std::unique_ptr<NUClear::Environment> environment) : Reactor(std::move(environment)), config{{}} {{
+        {className}::{className}(std::unique_ptr<NUClear::Environment> environment) : BehaviourReactor(std::move(environment)) {{
 
-            on<Configuration>("{className}.yaml").then([this](const Configuration& cfg) {{
+            on<Configuration>("{className}.yaml").then([this](const Configuration& config) {{
                 // Use configuration here from file {className}.yaml
-                this->log_level = cfg["log_level"].as<NUClear::LogLevel>();
+                this->log_level = config["log_level"].as<NUClear::LogLevel>();
             }});
```

Comment:
We used `cfg` rather than config here because often there is a config struct people use. And if both are named config it becomes trickier

Reply:
I updated the generated struct to be `cfg` since that's what we've been doing for a while now.